### ISSUE TITLE
create values for analysisType: haQTL, sQTL

### ISSUE THREAD

**Issue by avanlinden** [Sage-Bionetworks/sysbioDCCjsonschemas#133]
**For term:** analysisType

**Proposed value:** histone acetylation quantitative trait loci detection
- Definition: (can't find an ontology source, maybe a paper? abbreviated haQTL)
- Source: 

**Proposed value:** splicing quantitative trait loci detection
- Definition: (can't find an ontology source, abbreviated sQTL)
- Source:

_(Optional)_ Linked issues, contributor requests, or other information: <more info> 

These are for Ricardo's [SV_xQTL](https://github.com/Sage-Bionetworks/AD-DCC/issues/940#) analysis files. He has eQTL (expression quantitative trait loci detection, already in the dictionary) and pQTL (protein quantitative trait loci detection, already in the dictionary), as well as haQTL and sQTL which we need terms for. 

**To do:**
- [ ] data coordinator provides value, definition, and source in this issue, then assigns to relevant DCC curator
- [ ] curator has added value to [synapseAnnotations repo](https://github.com/Sage-Bionetworks/synapseAnnotations)

**Comment by avanlinden** [comment]
@pitviper6 Now I'm not sure whether we should add these keys as the abbreviated form ('haQTL') vs the full 'histone acetylation quantitative trait loci detection'. Right now, our various QTL values all take the **long** form. I really think these should all be parallel, so we either need to update our existing QTLs to be abbreviated or add these new keys as the long form. What do you think is better?

**Comment by pitviper6** [comment]
@avanlinden It depends on what most users would use to search. They all tend to use the abbreviated form in emails and such so I think it would make sense to use that.

How big of a project would it be to change our existing values in the dictionary and the existing annotations?

**Comment by avanlinden** [comment]
I don't think it would be too bad -- they're all on Analysis files, so there aren't too many. And `analysisType` is only a file annotation, not a value in the metadata files. Let's change the terms to the abbreviated version in the dictionary and then maybe I'll have Hieu update the annotations when she starts.

**Comment by pitviper6** [comment]
@avanlinden sounds good - I'll create the issues to create these
